doc/cascadia/profiles.schema.json

```diff
@@ -215,7 +216,7 @@
             "split": {
               "$ref": "#/definitions/SplitState",
               "default": "vertical",
-              "description": "The orientation to split the pane in, either vertical (think [|]) or horizontal (think [-])"
+              "description": "The orientation to split the pane in, either vertical (think [|]), horizontal (think [-]), or auto"
```

Comment:
Could you describe auto in the description too?

Reply:
Sure thing